src/SamplesApp/SamplesApp.UITests/Windows_UI_Xaml_Input/Capture_Tests.cs

```diff
@@ -21,34 +21,34 @@ public partial class Capture_Tests : SampleControlUITestBase
 	[ActivePlatforms(Platform.iOS, Platform.Android)] // This fails with unit test
 	[InjectedPointer(PointerDeviceType.Touch)]
 	public async Task TestSimple()
-		=> RunTest("Simple", TouchAndMoveOut);
+		=> await RunTest("Simple", TouchAndMoveOut);
```

Comment:
Did these tests not run before? I mean they appear as if they run in CI, but the Task that executes the core of the test wasn't executed.